src/main/java/de/zalando/ep/zalenium/aspect/HubAspect.aj

```diff
@@ -81,7 +81,7 @@ privileged public aspect HubAspect {
         handler.addServlet(LivePreviewServlet.class, "/grid/admin/live");
         handler.addServlet(ZaleniumConsoleServlet.class, "/grid/console");
         // We want resources to be at least 3 levels deep so that the prometheus servlet filter doesn't pick up all the individual resources
-        handler.addServlet(ZaleniumResourceServlet.class, "/grid/console/resources");
+        handler.addServlet(ZaleniumResourceServlet.class, "/grid/resources");
```

Comment:
I think we should move the resources out of `/grid` so that it doesn’t appear in `/metrics` at all.